ml-agents/tests/yamato/yamato_utils.py

```diff
@@ -45,6 +48,8 @@ def run_standalone_build(
         test_args += ["-logfile", "-"]
     if output_path is not None:
         test_args += ["--mlagents-build-output-path", output_path]
+    if scene_path is not None:
+        test_args += ["--mlagents-build-scene-path", scene_path]
```

Comment:
The whole PR make sense to me. Just wondering who is parsing this --mlagents-build-scene-path arg?

Reply:
https://github.com/Unity-Technologies/ml-agents/blob/master/Project/Assets/ML-Agents/Editor/Tests/StandaloneBuildTest.cs#L11
That would be the BuildStandalonePlayerOSX method in the StandaloneBuildTest.cs file.